design/gas/Design_gas.md

```diff
@@ -0,0 +1,72 @@
+# Obscuro Gas Design
+
+## Overview
+
+The revenue generation mechanism for Obscuro hinges on the collection of gas fees. Notably, as Obscuro operates as a layer 2 protocol, the cost of operations surpasses that of layer 1, since it has to cover layer 1 gas costs in addition to its operational expenses. Unlike layer 1 miners, who only shoulder operational costs, there's no expenditure prerequisite for block publication on Obscuro except for the static cost of stake.
```

Comment:
I'd add to the overview that the gas mechanics has typically multiple roles:
- revenue for node operators
- congestion management and prevention of denial of service
- price discovery mechanism for transactions

Reply:
Also would be good to add a quick summary here in how Ethereum and Optimism/Arbitrum solve these problems